localisation/wc_culture_buildings.csv

```diff
@@ -1,212 +1,212 @@
+ca_culture_wildhammer_1;Gryphon Aerie;;Greifenhorst;;;;;;;;;;;x
+ca_culture_wildhammer_2;Gryphon Aerie;;Greifenhorst;;;;;;;;;;;x
+ca_culture_wildhammer_3;Gryphon Aerie;;Greifenhorst;;;;;;;;;;;x
+ca_culture_wildhammer_4;Gryphon Aerie;;Greifenhorst;;;;;;;;;;;x
+ca_culture_wildhammer_desc;After the War of the Three Hammers, the Wildhammer dwarves found themselves residing in the heavily-wooded Hinterlands, where they befriended and tamed the local gryphons. Learning to use them as steeds, the Wildhammers soon became famous for their gryphon riders. These riders pride themselves as powerful airborne cavalry wielding enchanted stormhammers.;;Nach dem Krieg der drei H�mmer fanden sich die Wildhammerzwerge im waldreichen Hinterland wieder, wo sie sich mit den dortigen Greifen anfreundeten und sie z�hmten. Als sie lernten, sie als Reittiere zu nutzen, wurden die Wildhammer ber�hmt f�r ihre Greifenreiter. Diese Reiter r�hmen sich als m�chtige Luftkavallerie, die verzauberte Sturmh�mmer schwingt.;;;;;;;;;;;x
+ca_culture_bronzebeard_1;Siegeworks;;Belagerungswerkst�tten;;;;;;;;;;;x
+ca_culture_bronzebeard_2;Siegeworks;;Belagerungswerkst�tten;;;;;;;;;;;x
+ca_culture_bronzebeard_3;Siegeworks;;Belagerungswerkst�tten;;;;;;;;;;;x
+ca_culture_bronzebeard_4;Siegeworks;;Belagerungswerkst�tten;;;;;;;;;;;x
+ca_culture_bronzebeard_desc;Designed by gnomes and perfected by dwarves, this is the story of the siege engine. A replacement for the steam tank, the siege engine combines both ramming and shelling capabilities with heavy armoured plating and anti-air flak cannons. These metal beasts enable teams of dwarves to safely and effectively reduce enemy fortifications to powder.;;Von Gnomen entworfen und von Zwergen perfektioniert, dies ist die Geschichte der Belagerungsmaschine. Als Ersatz f�r den Dampfpanzer kombiniert die Belagerungsmaschine Ramm- und Gesch�tzf�higkeiten mit schwerer Panzerung und Flakgesch�tzen zur Luftabwehr. Diese metallenen Unget�me erm�glichen es Zwergenmannschaften, feindliche Befestigungen sicher und effektiv zu pulverisieren.;;;;;;;;;;;x
+ca_culture_pirate_1;Buccaneer Hold;;Seer�uberfestung;;;;;;;;;;;x
+ca_culture_pirate_2;Buccaneer Hold;;Seer�uberfestung;;;;;;;;;;;x
+ca_culture_pirate_3;Buccaneer Hold;;Seer�uberfestung;;;;;;;;;;;x
+ca_culture_pirate_4;Buccaneer Hold;;Seer�uberfestung;;;;;;;;;;;x
+ca_culture_pirate_desc;Vicous and bloodthirsty in their own right, these pirate buccaneers have nothing left to lose while spending every effort possible to loot and pillage the properties of those more fortunate than them. Whether it be by musket, bow or cutlass, the unlucky ones who cross their path end up feeding the fish.;;B�sartig und blutr�nstig an sich haben diese Seer�uber nichts mehr zu verlieren, w�hrend sie alles daran setzen, die Besitzt�mer der Gl�cklicheren zu pl�ndern und zu brandschatzen. Ob mit Muskete, Bogen oder Entermesser, die Ungl�cklichen, die ihren Weg kreuzen, enden als Futter f�r die Fische.;;;;;;;;;;;x
```

Comment:
Vicious was spelled incorrectly.
```suggestion
ca_culture_pirate_desc;Vicious and bloodthirsty in their own right, these pirate buccaneers have nothing left to lose while spending every effort possible to loot and pillage the properties of those more fortunate than them. Whether it be by musket, bow or cutlass, the unlucky ones who cross their path end up feeding the fish.;;B�sartig und blutr�nstig an sich haben diese Seer�uber nichts mehr zu verlieren, w�hrend sie alles daran setzen, die Besitzt�mer der Gl�cklicheren zu pl�ndern und zu brandschatzen. Ob mit Muskete, Bogen oder Entermesser, die Ungl�cklichen, die ihren Weg kreuzen, enden als Futter f�r die Fische.;;;;;;;;;;;x
```

Reply:
and there i thought this word had only one spelling mistake :)